plugin/src/main/groovy/grails/plugin/gormtools/RepositoryArtefactHandler.groovy

```diff
@@ -32,7 +32,7 @@ import static org.grails.io.support.GrailsResourceUtils.REGEX_FILE_SEPARATOR
 @CompileStatic
 class RepositoryArtefactHandler extends ArtefactHandlerAdapter {
 
-    static final String TYPE = "Repository"
+    public static final String TYPE = "Repository"
```

Comment:
why the public designation here?

Reply:
Actually wanted to access it from AST java class.
But actually it wasnt accessible anyway because of being in groovy folder.

Artefact handles have it public anyway so that if any class wants to use the type, they can access it.

Like DomainClassArtefactHandler.TYPE etc

eg. grailsApplication.getArtefact(name, type)

Reply:
not clear are all why this is needed, add a comment to code if it really is.